provider/gke/gke.go

```diff
@@ -48,7 +47,7 @@ type GKE struct {
 	// These are also used when the command requires some variables that are not provided by the deployment file.
 	DeploymentVars map[string]string
 	// DeploymentFile content after substituting the variables filename is used as the map key.
-	deploymentsContent map[string][]byte
+	deploymentsContent []provider.ResourceFile
```

Comment:
Using maps, the order of manifest files is not preserved.
For `-f b.yaml -f a.yaml`, we should process b.yaml first.
Also for `-f folder`, we should process in order of files present in the folder.

Reply:
yes good point , we still need to know the the filename somehow for the logging.

Reply:
yep good, point, it would be good to put a comment comment somewhere that when a folder is provided the files are  applied in alphabetical order and when multiply files are provided by the -f flag these are also applied by their order.